src/java.base/share/classes/sun/text/RuleBasedBreakIterator.java

```diff
@@ -70,7 +70,7 @@
  * expression defines a set of characters (the &quot;<em>ignore characters</em>&quot;) that
  * will be transparent to the BreakIterator.&nbsp; A sequence of characters will break the
  * same way it would if any ignore characters it contains are taken out.&nbsp; Break
- * positions never occur befoer ignore characters.</p>
+ * positions never occur before ignore characters.</p>
```

Comment:
"before ignored characters"

Reply:
I believe it's the name of a concept, so I will leave it as is:

> There is one special substitution.  If the description defines a substitution called "<ignore>", the expression must be a [] expression, and the expression defines a set of characters (the "ignore characters") that will be transparent to the BreakIterator.

Reply:
Yes, that is correct.